app.rb

```diff
@@ -8,6 +8,10 @@
 class CondaAPI < Sinatra::Base
   scheduler = Rufus::Scheduler.new
 
+  configure :production, :development do
+    enable :logging
+  end
```

Comment:
We did this in another app to enable request logging (requests aren't currently being logged in GCP), so I'm giving this a shot here.